api/src/main/java/jakarta/data/repository/Repository.java

```diff
@@ -537,38 +538,8 @@
      * available Jakarta Data provider that supports the type of entity
      * annotation that is present on the repository's entity class.
      */
-    static final String ANY_PROVIDER = "";
+    String ANY_PROVIDER = "";
 
-    /**
-     * Value for the {@link dataStore} attribute that indicates that the
-     * Jakarta Data provider should choose a default data store to use.
-     * The default data store might require additional vendor-specific
-     * configuration, depending on the vendor.
-     */
-    static final String DEFAULT_DATA_STORE = "";
-
-    /**
-     * <p>Optionally indicates the data store to use for the repository.</p>
-     *
-     * <p>The Jakarta Data specification does not define a full configuration
-     * model, and relies upon the Jakarta Data providers to provide configuration.
-     * This value serves as an identifier linking to vendor-specific configuration
-     * for each Jakarta Data provider to interpret in a vendor-specific way.</p>
-     *
-     * <p>For some Jakarta Data providers, this value could map directly to an
-     * identifier in vendor-specific configuration. For others, this value could
-     * map to a base configuration path that forms a configuration hierarchy,
-     * such as in MicroProfile Config, or possibly Jakarta Config in a future
-     * version of this specification. For providers backed by Jakarta Persistence,
-     * it might point to a {@code jakarta.annotation.sql.DataSourceDefinition} name
-     * or a {@code javax.sql.DataSource} JNDI name or resource reference,
-     * or other vendor-specific configuration.</p>
-     *
-     * <p>The default value of this attribute is {@link #DEFAULT_DATA_STORE}.</p>
-     *
-     * @return the name of a data store or {@link #DEFAULT_DATA_STORE}.
-     */
-    String dataStore() default DEFAULT_DATA_STORE;
```

Comment:
This is not specific to Jakarta Persistence.  It applies to every type of backend where there can be multiple, which is all types I'm aware of.

Reply:
Usually, we're using convention over configuration allied with The Twelve-Factor App:
https://12factor.net/
E.g.:

* [Spring](https://docs.spring.io/spring-boot/docs/current/reference/html/application-properties.html#appendix.application-properties.data-migration)
* [Micronaut](https://guides.micronaut.io/latest/micronaut-jpa-hibernate-gradle-java.html#jpa-configuration)
* [Quarkus](https://quarkus.io/guides/config)

Reply:
I don't follow this argument at all with respect to what it has to do with JPA being the deciding factor in whether or not you are able to use multiple databases.